src/test/java/games/strategy/engine/data/annotations/ValidateAttachmentsTest.java

```diff
@@ -364,11 +353,10 @@ private static String validateAttachment(final Class<? extends IAttachment> claz
       } else if (!Modifier.isAbstract(clazz.getModifiers())) {
         // check the symmetry of regular setters
         @SuppressWarnings("unused")
-        String method = null;
+        final String method = null;
```

Comment:
Looks like all references to this variable have been removed, so the declaration itself is no longer necessary.